### PR TITLE
Custom error ("404 not found") page

### PR DESCRIPTION
**Pull request by viskey98**
Fixes issue #66

### CODE REVIEW

**Comment by viskey98** [comment]
@afeena full work has been done on this issue , without selenium. Please look into it and suggest any changes if required. :)

File: clone.py [hunk 1]
```diff
@@ -145,10 +146,10 @@ def _make_filename(self, url):
             content_type = None
             try:
                 with aiohttp.Timeout(10.0):
-                    response = await session.get(current_url)
-                    content_type = response.content_type
-                    data = await response.read()
-
+                        response = await session.get(current_url, headers={'Accept': 'text/html'})
```
Comment (glaslos) [inline clone.py]: broken indentation

File: snare.py [hunk 2]
```diff
@@ -216,16 +217,32 @@ def create_data(self, request, response_status):
         content = None
         status_code = 200
         headers = {}
-
+        p = re.compile('/+')
+        requested_name = p.sub('/',requested_name)
```
Comment (glaslos) [inline snare.py]: missing space and add a comment what you are doing here

**Comment by viskey98** [comment]
@glaslos I have made the required changes suggested by you, please look into it.  :)

**Pull request merged**